analysis/src/CompletionBackEnd.ml

```diff
@@ -711,6 +711,16 @@ let detail name (kind : Completion.kind) =
   | FileModule _ -> "file module"
   | Field ({typ}, s) -> name ^ ": " ^ (typ |> Shared.typeToString) ^ "\n\n" ^ s
   | Constructor (c, s) -> showConstructor c ^ "\n\n" ^ s
+  | PolyvariantConstructor {name; payload} -> (
+    "#" ^ name
+    ^
+    (* TODO: I get the feeling there's a more clever way to do this printing...
```

Comment:
This code looks fine. No need to be too clever. It just reflects the internal representation of poly variants.